tests/test_verilog/test_inline.py

```diff
@@ -103,7 +103,8 @@ class Main(m.Circuit):
             ready_out=delay.inner_delay.inner_inner_delay.INPUT[1].ready,
         )
 
-    m.compile(f"build/test_inline_tuple", Main, output="mlir")
+    m.compile(f"build/test_inline_tuple", Main, output="mlir",
```

Comment:
can we have this parametrized for both flatten_all_tuples = False, True?

Reply:
Don't think this is necessary since it's going to be deprecated for inline_verilog2?

Reply:
(The test is parametrized for inline_verilog2)